src/PhpSpec/Formatter/FatalErrorWriter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Formatter;
+
+use PhpSpec\Console\IO;
+use PhpSpec\Message\CurrentExample;
+
+final class FatalErrorWriter implements FatalPresenterInterface
```

Comment:
So I've been thinking... :-)

This should maybe be the responsibility of the actual selected Formatter. For instance the JUnit formatter might choose to do something different to the progress formatter in the instance of a fatal.

This would require making a new Formatter interface with an `onFatal` and handling the cases where users have a different formatter that uses the old Interface.

This solution where the Formatter and the FatalPresenter are separated out probably doesn't work for all the different formatters users might select.